node/lib/openshift-origin-node/model/node.rb

```diff
@@ -366,6 +366,13 @@ def self.node_utilization
           end
         end
 
+        # fact for available disk space on a node, considered in gear moves to ensure minimum buffer of free space on nodes
+        mountpoint = self.get_gear_mountpoint
+        cmd = "df #{mountpoint} | awk -F' ' 'NR == 2 {print $4}'"
+        df, stderr, rc = Utils.oo_spawn(cmd)
+        raise NodeCommandException.new "Error: #{stderr} executing command #{cmd}" unless rc == 0
+        res['node_disk_free'] = df
```

Comment:
`df` has a newline character at this point (you can see this if you view `/opt/rh/ruby193/root/etc/mcollective/facts.yaml`); use `df.chomp` instead to remove it.  Alternatively, use `to_i` here (`to_i` ignores whitespace) and store the fact as an integer instead of a string.